changelog.md

```diff
@@ -3,7 +3,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-# [Unreleased](https://github.com/MyPureCloud/genesys-cloud-webrtc-sdk/compare/v8.0.5...HEAD)
+# [Unreleased](https://github.com/MyPureCloud/genesys-cloud-webrtc-sdk/compare/v8.0.6...HEAD)
+# [v8.0.6](https://github.com/MyPureCloud/genesys-cloud-webrtc-sdk/compare/v8.0.5...v8.0.6)
+### Fixed
+* Bump streaming client
```

Comment:
Really not a big deal but can you tie this to a ticket? There may not be one or it may be part of the one below, PCM-2089.

Reply:
Or it might be PCM-2093

Reply:
Yeah, good point.